schemas/ns10_cache_entry.fbs

```diff
@@ -0,0 +1,13 @@
+
+file_identifier "ns10";
+
+/// pylint: skip-file
+table CacheEntry {
+        key:string;            // key for this entry
+        time:double;           // time when this entry was set
+        ttl:double;            // time to live
+        expired:bool = false;  // is this entry already expired (manually or using ttl)
+        value:string;          // value for the key
```

Comment:
Can there also be arrays encoded here as string? Or even (json-style) objects? Or only primitives?